### PR TITLE
{bio}[foss/2022a] duplex-tools v0.3.3 w/ Python 3.10.4

### PR DESCRIPTION
**Pull request by PetrKralCZ**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/d/duplex-tools/duplex-tools-0.3.3-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+_minimap2_ver = '2.24'
+dependencies = [
+    ('Python', '3.10.4'),
+    ('SciPy-bundle', '2022.05'),
+    ('matplotlib', '3.5.2'),
+    ('edlib', '1.3.9'),
+    ('minimap2', _minimap2_ver),
+    ('python-parasail', '1.3.3'),
+    ('Pysam', '0.20.0'),
```
Comment (boegel) [inline easybuild/easyconfigs/d/duplex-tools/duplex-tools-0.3.3-foss-2022a.eb:25]: @PetrKralCZ please stick to Pysam 0.19.1

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by PetrKralCZ** [comment]
Test report by @PetrKralCZ
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node4012.donphan.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz, 1 x NVIDIA NVIDIA A2, 530.30.02, Python 3.6.8
See https://gist.github.com/PetrKralCZ/3e765673eae315b74d6263628d5cd9e7 for a full test report.

**Comment by smoors** [comment]
@boegelbot please test @ jsc-zen2

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @PetrKralCZ!

**Pull request merged**